src/Components/Stack/index.tsx

```diff
@@ -17,11 +17,10 @@ const Stack: React.FunctionComponent<IStack.IProps> = ({ stackId }) => {
       <div className="stack">
         <div className="stack-main">
           <div className="title-area">
-            {stack.order ? (
-              <span className="order-redesigned">
-                <p>{stack.order}</p>
-              </span>
-            ) : null}
+            <span className="order-redesigned">
+              <p>{stack.order ? stack.order + 1 : 1}</p>
```

Comment:
Sometimes we don't want to display the stack order. Change the check to `displayOrder && typeof stack.order === 'number' ? <span className ... /> : null`. You will need to create a new props called `displayOrder`.